doc/neps/nep-0055-string_dtype.rst

```diff
@@ -1060,16 +1064,18 @@ a "medium"-length string and the size requires only one byte in the arena
 allocation. An arena string larger than 255 bytes would need 8 bytes in the
 arena to store the size in a ``size_t``. The only flag set indicates that this
```

Comment:
This is a bit confusing, with two sentences "the only flag set"

Reply:
Oops, remnant of the previous version. Thank you for catching.